File: wms/src/main/webapp/system/app/model/bdef/pntdef_PrinterGrpgatherModel.js
/**
 * 模块名称：打印机群组与打印机组关系维护
 * 模块编码：1T01
 * 创建：hcx 
 */
Ext.define('cms.model.bdef.pntdef_PrinterGrpgatherModel', {
    extend: 'Ext.data.Model',
    idProperty:'enterpriseNo,warehouseNo,printerGroupNo',
    fields: [
        {name:'enterpriseNo'},
		{name:'warehouseNo'},
		{name:'grpgatherNo'},
		{name:'grpgatherName'},
		{name:'rgstName'},
		{name:'rgstDate'},
		{name:'updtName'},
		{name:'updtDate'}
    ]
});

File: model/src/main/java/com/sealinkin/bdef/model/Pntdef_PrinterGrpgatherModel.java
package com.sealinkin.bdef.model;

import com.sealinkin.bdef.po.Pntdef_PrinterGrpgather;

public class Pntdef_PrinterGrpgatherModel extends Pntdef_PrinterGrpgather{

	private static final long serialVersionUID = 1L;
	
	private String enterpriseNo;
	private String warehouseNo;
	private String grpgatherNo;
	public String getWarehouseNo() {
		return warehouseNo;
	}
	public void setWarehouseNo(String warehouseNo) {
		this.warehouseNo = warehouseNo;
	}
	
	public String getEnterpriseNo() {
		return enterpriseNo;
	}
	public void setEnterpriseNo(String enterpriseNo) {
		this.enterpriseNo = enterpriseNo;
	}
	public String getGrpgatherNo() {
		return grpgatherNo;
	}
	public void setGrpgatherNo(String grpgatherNo) {
		this.grpgatherNo = grpgatherNo;
	}
}
